applications/migrations/0028_auto_20191214_2351.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
The two migrations to applications may messed up how it's setup on the server side

Reply:
Everything else looks good though :)

Reply:
It should still be okay on the server rn right? Since it hasn't been merged yet

Reply:
Yeah it should still be ok but we should delete the application migrations before merging the workshop changes